nuxeo-filters-behavior.html

```diff
@@ -132,8 +132,8 @@
     /**
      * Checks if the document has a main blob.
      */
-    hasContent(doc) {
-      return (doc && doc.properties) ? doc.properties['file:content'] : false;
+    hasContent(doc, xpath) {
+      return (doc && doc.properties) ? this.get((xpath || 'file:content'), doc.properties) : false;
```

Comment:
nit: round braces around xpath are redundant
```suggestion
      return (doc && doc.properties) ? this.get(xpath || 'file:content', doc.properties) : false;
```